examples/petset/README.md

```diff
@@ -0,0 +1,146 @@
+# MongoDB Replication Example Using a PetSet
+
+This [MongoDB replication](https://docs.mongodb.com/manual/replication/) example
+uses a [PetSet](http://kubernetes.io/docs/user-guide/petset/) to manage replica
+set members with scalable persistent storage in
+[OpenShift](https://www.openshift.org/).
+
+It is supported by an example [OpenShift
+template](https://docs.openshift.org/latest/dev_guide/templates.html) and
+scripts that automate replica set initiation, baked in the multiple versions of
+the MongoDB images built from this source repository:
+
+- [openshift/mongodb-24-centos7](https://hub.docker.com/r/openshift/mongodb-24-centos7/)
+- [centos/mongodb-26-centos7](https://hub.docker.com/r/centos/mongodb-26-centos7/)
+- [centos/mongodb-32-centos7](https://hub.docker.com/r/centos/mongodb-32-centos7/)
+- and RHEL variants
+
+## Getting Started
+
+You will need an OpenShift cluster where you can deploy a template. If you don't
+have an existing OpenShift installation yet, the easiest way to get started and
+try out this example is using the
+[`oc cluster up`](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md)
+command.
+
+Assuming you have the `oc` tool, are logged in, and in the context of a project
+where you want to create a MongoDB cluster, run `oc new-app` passing the
+template file as an argument (`oc new-app` understands URLs too):
+
+```bash
+oc new-app https://raw.githubusercontent.com/sclorg/mongodb-container/master/examples/petset/mongodb-petset.yaml
```

Comment:
Yes, it requires pre-provisioned PVs. The alternative is to use an automatic provision (another alpha feature), but that requires changes to the template.

I'll update this section later to mention PVs.